source/Player_Setup.py

```diff
@@ -0,0 +1,25 @@
+from CardGames import *
+
+
+def Player_Setup(Playerlist):
+
+    #Finding out number of players with an error for invalid inputs
+    NumOfPlayers = input("How many people are playing? ")
+
+
+    #A loop for creating various players with name and money amounts
+    try: 
```

Comment:
I like this loop! It's clean and concise.

Reply:
les go Joooe. Everythn looks good